docs/source/en/model_doc/fastspeech2_conformer.md

```diff
@@ -10,97 +10,91 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
+```py
+from transformers import AutoModel, AutoTokenizer
 import soundfile as sf
 
-tokenizer = FastSpeech2ConformerTokenizer.from_pretrained("espnet/fastspeech2_conformer")
+tokenizer = AutoTokenizer.from_pretrained("espnet/fastspeech2_conformer")
+model = AutoModel.from_pretrained("espnet/fastspeech2_conformer")
+
 inputs = tokenizer("Hello, my dog is cute.", return_tensors="pt")
 input_ids = inputs["input_ids"]
 
-model = FastSpeech2ConformerModel.from_pretrained("espnet/fastspeech2_conformer")
 output_dict = model(input_ids, return_dict=True)
 spectrogram = output_dict["spectrogram"]
 
+from transformers import FastSpeech2ConformerHifiGan
 hifigan = FastSpeech2ConformerHifiGan.from_pretrained("espnet/fastspeech2_conformer_hifigan")
 waveform = hifigan(spectrogram)
 
 sf.write("speech.wav", waveform.squeeze().detach().numpy(), samplerate=22050)
 ```
 
-3. Run inference via the Transformers modelling code with the model and hifigan combined
+</hfoption>
+</hfoptions>
+
+For a more streamlined experience, you can use the combined model with the HiFi-GAN vocoder in a single step:
 
 ```python
 from transformers import FastSpeech2ConformerTokenizer, FastSpeech2ConformerWithHifiGan
 import soundfile as sf
 
 tokenizer = FastSpeech2ConformerTokenizer.from_pretrained("espnet/fastspeech2_conformer")
+model = FastSpeech2ConformerWithHifiGan.from_pretrained("espnet/fastspeech2_conformer_with_hifigan")
+
 inputs = tokenizer("Hello, my dog is cute.", return_tensors="pt")
-input_ids = inputs["input_ids"]
 
-model = FastSpeech2ConformerWithHifiGan.from_pretrained("espnet/fastspeech2_conformer_with_hifigan")
-output_dict = model(input_ids, return_dict=True)
+output_dict = model(inputs["input_ids"], return_dict=True)
 waveform = output_dict["waveform"]
 
 sf.write("speech.wav", waveform.squeeze().detach().numpy(), samplerate=22050)
 ```
 
-4. Run inference with a pipeline and specify which vocoder to use
-```python
-from transformers import pipeline, FastSpeech2ConformerHifiGan
-import soundfile as sf
-
-vocoder = FastSpeech2ConformerHifiGan.from_pretrained("espnet/fastspeech2_conformer_hifigan")
-synthesiser = pipeline(model="espnet/fastspeech2_conformer", vocoder=vocoder)
-
-speech = synthesiser("Hello, my dog is cooler than you!")
-
-sf.write("speech.wav", speech["audio"].squeeze(), samplerate=speech["sampling_rate"])
-```
+## Notes
 
+- FastSpeech2Conformer uses a non-autoregressive architecture, making it significantly faster than autoregressive TTS models like Tacotron.
+- The model combines a duration predictor, pitch predictor, and energy predictor to control various aspects of speech generation.
+- For optimal audio quality, make sure to use the matching HiFi-GAN vocoder (`espnet/fastspeech2_conformer_hifigan`).
+- To improve generation speed, consider using the combined model (`FastSpeech2ConformerWithHifiGan`) which handles both spectrogram generation and vocoding in a single step.
```

Comment:
```suggestion
- Use the matching [HiFi-GAN vocoder](https://huggingface.co/espnet/fastspeech2_conformer_hifigan) for optimal audio quality.
- Use the combined model, [`FastSpeech2ConformerWithHiFiGan`], for faster generation speed since it handles both spectrogram generation and vocoding in a single step.
- You could also use the [`FastSpeech2ConformerModel`] and [`FastSpeech2ConformerHiFiGan`] separately as shown below.
   <add code snippet here>
```